changelogs/fragments/385-vrf-support-for-ovpn-server

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - api_info, api_modify - Since RouterOS 7.17 VRF is supported for OVPN server.
+    It now supports multiple entries and single_value parameter is now obsolete. This demands for a versioned API.
+    https://github.com/ansible-collections/community.routeros/pull/383
```

Comment:
```suggestion
  - api_info, api_modify - since RouterOS 7.17 VRF is supported for OVPN server.
    It now supports multiple entries, while ``api_modify`` so far only accepted a single entry.
    The ``interface ovpn-server server`` path now allows multiple entries
    on RouterOS 7.17 and newer
    (https://github.com/ansible-collections/community.routeros/pull/383).
```